ide-support/revsaveasandroidstandalone.livecodescript

```diff
@@ -684,9 +684,14 @@ private command revSaveAsMobileStandaloneMain pStack, pApkFile, pTarget, pSettin
       end if
       put tBuildFolder & slash & "classes_app/" & tCompiledClassFilePath & "/AppProvider.class" & return after tBuiltClassFiles
       
+      set the itemdel to slash
+      repeat for each line tJarFile in pSettings["jarFiles"]
+         put url ("binfile:" & tJarFile) into url ("binfile:" & tArmLibsBuildFolder & slash & item -1 of tJarFile)
```

Comment:
I think this should be `tLibsBuildFolder` as `tArmLibsBuildFolder` should be for compiled arm libraries.

Reply:
Haha, damn-  that's what I did originally then changed it